lumigator/python/mzai/evaluator/entrypoint.py

```diff
@@ -51,7 +51,7 @@ def evaluate(self, config: EvaluationJobConfig) -> EvaluationResult:
         The underlying evaluation framework is determined by the configuration type.
         """
         print("evaluating...")
-        logger.info("evaluating config: %s", config)
+        logger.info(f"evaluating config: {config}")
```

Comment:
We'll need to log with print statements here since these logs are surfaced through Ray, which uses print for default

Reply:
Ah ok, that's good to know!
Il will fix the print adding the % then